### PR TITLE
Add 'none' consistency model for fully supervised

### LINKED ISSUE #19: Learning rate for fully supervised baseline is wrong
The flags for the fully-supervised baselines choose `mean_teacher` for the consistency model, intended as a no-op because the consistency weight is set to zero. But the hparams for Mean Teacher dictate a different learning rate than the best one we found for fully-supervised (and also the one we state we used in the paper).

### PR DESCRIPTION
**Pull request by craffel**
We were using the wrong learning rate for fully-supervised because we were setting the consistency model to `mean_teacher` with `max_cons_multiplier=0`. This ended up causing a hyperparameter override to set the wrong learning rate. This PR adds a consistency model `none` which explicitly avoids using any consistency model and also allows us to have a separate hyperpamater override dict for fully-supervised learning (hopefully preventing this kind of issue in the future).

Fixes #19

### CODE REVIEW

**Review by avital — APPROVED** [review]
LGTM
